_licenses/eupl-1.1.txt

```diff
@@ -0,0 +1,345 @@
+﻿---
+title: European Union Public License 1.1
+nickname: EUPL-1.1
+tab-slug: eupl-v1.1
+redirect_from: /licenses/eupl-v1.1/
+family: EUPL
+featured: false
+source: https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence-eupl-v11
+
+description: The “European Union Public Licence” (EUPL) is the first European Free/Open Source Software (F/OSS) licence. It has been created on the initiative of the European Commission. It is now approved by the European Commission in 22 official languages of the European Union.
```

Comment:
I still don't see how "first European F/OSS license" is true given CeCILL. I suggest something like

description: The “European Union Public Licence” (EUPL) is a copyleft free/open source software license created on the initiative of and approved by the European Commission in 22 official languages of the European Union.